src/main/java/io/deephaven/csv/CsvSpecs.java

```diff
@@ -117,6 +117,34 @@ public interface Builder {
          */
         Builder headerValidator(Predicate<String> headerValidator);
 
+        /**
+         * True if the input is organized into fixed width columns rather than delimited by a delimiter.
+         */
+        Builder hasFixedWidthColumns(boolean hasFixedWidthColumns);
```

Comment:
What do we expect will happen if `ignoreSurroundingSpaces() == false`? Is that a legit configuration? Should we disallow it?

What about quote / trim behavior? I'm assuming that `"` (or quote char) inside the fixed width _are_ included in the resulting string. In which case, maybe we want to assert that if we are in fixed width mode, `quote()` should not be set (or, at least, not set differently than the default).

Mainly, I wonder if we need to consider the scope of CsvSpecs wrt hasFixedWidthColumns; I'm happy to err on the side of being more strict.

Reply:
`ignoreSurroundingSpaces() == false` is legit, though perhaps not very useful.  Now tested in `CsvReaderTest#fixedColumnsMayIncludeOrExcludeSurroundingSpaces`

quote / trim / etc now validated for only being set in the right mode. See `CsvReaderTest#checkParametersIncompatibleWithFixedWidthMode` and its partner `CsvReaderTest#checkParametersIncompatibleWithDelimitedMode`